recipe/meta.yaml

```diff
@@ -13,32 +13,39 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
 
 requirements:
   host:
     - python
+    - pip
     - setuptools
-    - pytest-runner
+    - wheel
   run:
     - python
     - six
 
 test:
+  requires:
+    - pip
   imports:
     - python_utils
     - python_utils.converters
     - python_utils.formatters
+  commands:
+    - pip check
 
 about:
   home: https://github.com/WoLpH/python-utils
   license: BSD-3-Clause
   license_family: BSD
   summary: 'Python Utils is a collection of small Python functions and classes which make common patterns shorter and easier.'
   description: |
-    Python Utils is a collection of small Python functions and classes which make common patterns shorter and easier. It is by no means a complete collection but it has served me quite a bit in the past and I will keep extending it.
-    One of the libraries using Python Utils is Django Utils.
+    Python Utils is a collection of small Python functions and classes which make common patterns shorter
+    and easier. It is by no means a complete collection but it has served me quite a bit in the past and
+    I will keep extending it. One of the libraries using Python Utils is Django Utils.
   doc_url: http://python-utils.readthedocs.io/
```

Comment:
We might have to use `https` instead of `http` on the `doc_url` 

https://github.com/AnacondaRecipes/python-utils-feedstock/blob/06feb72d273facba108259f8779a44e4d6b48bed/recipe/meta.yaml#L47

Reply:
good catch